### DISCUSSION TITLE
IP Cam Livestream

### DISCUSSION THREAD

**Discussion by Tom54497** [Zefau/ioBroker.jarvis#1634] [Ideas / Feature Requests]
Würde mir wirklich endlich mal eine vernünftige Einbindung von Livestreams wünschen. als ONVIf und/oder RTSP Stream. Die bisherige Aktualisierung von Standbildern bereitet mangels Aktualisierung des Bildes manche Probleme. Mal geht es, mal nicht! Besten Dank.

**Comment by renne94** [comment]
Ja, das wäre nice.

**Comment by Zefau** [comment]
Bitte Feature Requests immer als Issue anlegen (sofern es noch keins gibt) und hoch-voten. Feature werden nach User-Votes umgesetzt.

Das Feature Request https://github.com/Zefau/ioBroker.jarvis/issues/773 entspricht etwa der Anfrage hier.

Eine aktuelle Reihenfolge anhand der Votes seht ihr unter https://github.com/Zefau/ioBroker.jarvis/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions+label%3A%22%3Astar2%3A+feature%22
